tests/e2e/specs/price-benchmark/price-benchmark-suggestions.test.js

```diff
@@ -90,36 +90,21 @@ test.describe( 'Price Benchmark Page', () => {
 		} );
 
 		test( 'Does not render the chart if Market Insights not enabled for the account.', async () => {
-			await priceBenchmarkPage.fulfillPriceBenchmarkSuggestions(
+			await priceBenchmarkPage.fulfillPriceBenchmarkSuggestions( [] );
+
+			await priceBenchmarkPage.fulfillPriceBenchmarkSummary(
 				{
-					message: {
-						error: {
-							code: 403,
-							message:
-								'Market Insights not enabled for account 5330359695. For more information check https://support.google.com/merchants/answer/9625913',
-							errors: [
-								{
-									message:
-										'Market Insights not enabled for account 5330359695. For more information check https://support.google.com/merchants/answer/9625913',
-									domain: 'global',
-									reason: 'forbidden',
-								},
-							],
-							status: 'PERMISSION_DENIED',
-							details: [
-								{
-									'@type':
-										'type.googleapis.com/google.rpc.ErrorInfo',
-									reason: 'forbidden',
-									domain: 'global',
-								},
-							],
-						},
-					},
+					message: 'Forbidden.',
```

Comment:
@mukeshpanchal27 The condition which checks for `message.error.code === 403` in `actions.js` will never be met because you are not providing any error.code in the payload.